extensions/widgets/treeview/notes/feature-scroll_into_view.md

```diff
@@ -0,0 +1,19 @@
+# Properties
+
+The tree view widget now has the ability to scroll the selected row into 
```

Comment:
@bwmilby I think you should discuss the new property `hiliteNewElement` here also.

Reply:
Added property to the note.